examples/src/main/scala/app/Main.scala

```diff
@@ -0,0 +1,35 @@
+package app
+
+import io.circe.generic.auto._
+import io.circe.syntax._
+import zhttp.domain.http.HttpApp
+import zhttp.domain.http.model.Method.GET
+import zhttp.domain.http.model.Path.{/, Root}
+import zhttp.domain.http.model.Response
+import zhttp.service.netty.EventLoopGroup
+import zhttp.service.netty.server.{Server, ServerChannelFactory}
```

Comment:
Too much noise in imports. Can we reduce it to just a one line import from zhttp?